examples/permissioned-token/scripts/4-setup-newpermissions-app.js

```diff
@@ -0,0 +1,27 @@
+const accounts = require('./common/accounts');
```

Comment:
can we change filename to `4-setup-new-permissions-app.js` ?

Reply:
yes